lib/hanami/model/migrator/postgres_adapter.rb

```diff
@@ -109,7 +109,7 @@ def modified_message(original_message)
           case original_message
           when /already exists/
             DB_CREATION_ERROR
-          when /does not exist/
+          when /database ".*" does not exist/
```

Comment:
I think that `+` will be better here. Because `*` is about 0 or more and `+` - 1 and more

Reply:
Thank you. I fixed this.